gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -194,7 +190,7 @@ def _get_contigs_pysam(self, contigs):
       result['id'] = item[0]
       result['length'] = item[1].length
       results[item[0]] = result
-    return dict(results.items())
+    return results
 
   def _get_samples_pysam(self, sample_line):
     sample_tags = sample_line.split('\t')
```

Comment:
We should explain where this const 9 is coming from, either add a comment or use this variable:
```
default_items_num = len(LAST_HEADER_LINE_PREFIX.split('\t')
if len(sample_tags) > default_items_num:
  return sample_tags[(default_items_num):]
```
Also shouldn't this be `> 8`? `LAST_HEADER_LINE_PREFIX` has 8 fields.

Reply:
When there are no samples, there are 8 fields. When there are samples, there is also FORMAT field that describes call formats.

Used your suggestion, added +1, added a comment for explanation.